Source/WebCore/rendering/mathml/RenderMathMLFraction.cpp

```diff
@@ -259,10 +259,7 @@ void RenderMathMLFraction::layoutBlock(bool relayoutChildren, LayoutUnit)
     LayoutPoint denominatorLocation(borderAndPaddingLeft + denominator().marginLeft() + horizontalOffset(denominator(), element().denominatorAlignment()), verticalOffset);
     denominator().setLocation(denominatorLocation);
 
-    if (numerator().isOutOfFlowPositioned())
-        numerator().containingBlock()->insertPositionedObject(numerator());
-    if (denominator().isOutOfFlowPositioned())
-        denominator().containingBlock()->insertPositionedObject(denominator());
+    handlePositionedAndFloatingChildren();
```

Comment:
I would move this generic thing just before the fraction-specific layout starts (before the `numerator().layoutIfNeeded()` call).